### PR TITLE
EEA branch for immigration status question

### PR DESCRIPTION
**Pull request by lsalivonaite-transform**
[https://trello.com/c/PLMA3iMK/999-apply-review-options-on-visa-or-immigration-status-page-for-eea-candidates](https://trello.com/c/PLMA3iMK/999-apply-review-options-on-visa-or-immigration-status-page-for-eea-candidates)

Created a branch that shows 'EU settled status' or 'EU pre-settled status' if the nationality is European.
Screenshot 2023-12-01 at 14 20 52

And removed the EEA radio options from other nationalities.

### CODE REVIEW

File: app/views/details/personal-information/immigration-status.html [hunk 1]
```diff
@@ -43,9 +44,38 @@ <h1 class="govuk-heading-l">{{ title }}</h1>
...
   items: [
     {
-      value: "EU settled and pre-settled status",
-      text: "EU settled and pre-settled status"
+      value: "EU settled status",
+      text: "EU settled status"
     },
+    {
+      value: "Pre-settled status",
+      text: "Pre-settled status"
```
Comment (MaeveRoseveare) [inline app/views/details/personal-information/immigration-status.html:52]:
```suggestion
      text: "EU pre-settled status"
```

**Review by MaeveRoseveare — APPROVED** [review]
Looks good! For the dev ticket we should specify on the summary list the answer should populate  "Immigration status' and with whatever they write in the 'Other' box, instead of just 'Other'.

### Prototype:
Screenshot 2023-12-07 at 15 44 39

This is already the functionality in prod, but worth mentioning in the ticket for clarity incase they think we changed it.

### Production:
Screenshot 2023-12-07 at 15 47 04

**Comment by lsalivonaite-transform** [comment]
> Looks good! For the dev ticket we should specify on the summary list the answer should populate "Immigration status' and with whatever they write in the 'Other' box, instead of just 'Other'.
> 
> ### Prototype:
> Screenshot 2023-12-07 at 15 44 39
> This is already the functionality in prod, but worth mentioning in the ticket for clarity incase they think we changed it.
> 
> ### Production:
> Screenshot 2023-12-07 at 15 47 04

Should the label be 'immigration status' for non-EEA or 'visa or immigration status'?

**Comment by MaeveRoseveare** [comment]
> Should the label be 'immigration status' for non-EEA or 'visa or immigration status'?

It should say 'Immigration status'

**Comment by MylesJarvis** [comment]
> > Should the label be 'immigration status' for non-EEA or 'visa or immigration status'?
> 
> I think EEA it should be 'Visa or immigration status' and EEA just 'immigration status'

Prod currently uses 'Immigration status' across both. Your comment above @MaeveRoseveare mentions EEA twice there - which are you recommending for non-EEA?

**Comment by MaeveRoseveare** [comment]
> Prod currently uses 'Immigration status' across both. Your comment above @MaeveRoseveare mentions EEA twice there - which are you recommending for non-EEA?

Oh god sorry! I've edited my original comment now, so: I think non-EEA it should be 'Visa or immigration status' and EEA just 'Immigration status'

**Pull request merged**